### PR TITLE
add new facet for earliest accessioned date

### PR DESCRIPTION
**Pull request by peetucket**

### CODE REVIEW

File: app/models/concerns/document_date_concern.rb [hunk 1]
```diff
@@ -3,13 +3,14 @@
...
-  FIELD_REGISTERED_DATE       = :registered_dttsim
-  FIELD_LAST_ACCESSIONED_DATE = :accessioned_latest_dttsi
-  FIELD_LAST_PUBLISHED_DATE   = :published_latest_dttsi
-  FIELD_LAST_SUBMITTED_DATE   = :submitted_latest_dttsi
-  FIELD_LAST_DEPOSITED_DATE   = :deposited_latest_dttsi
-  FIELD_LAST_MODIFIED_DATE    = :modified_latest_dttsi
-  FIELD_LAST_OPENED_DATE      = :opened_latest_dttsi
+  FIELD_REGISTERED_DATE           = :registered_dttsim
+  FIELD_LAST_ACCESSIONED_DATE     = :accessioned_latest_dttsi
+  FIELD_EARLIEST_ACCESSIONED_DATE = :accessioned_earliest_dttsi
```
Comment (peetucket) [inline app/models/concerns/document_date_concern.rb]: note...this is the only line added, everything else is just re-aligning the spacing

**Comment by jcoyne** [comment]
Argo is pretty slow to draw pages, and I've heard that a lot of this is because the faceting is expensive for solr.  Do we know what the impact of adding this will be?  How much performance degradation are we willing to accept in order to add this feature?

**Comment by peetucket** [comment]
I am not sure what the impact will be of adding yet more more facet -- we could do some smoke testing on argo-stage and have @blalbrit provide his opinion.

**Comment by blalbrit** [comment]
Happy to smoke test before we go all out on this. We would definitely want to add this to the facets that are buried behind the "more facets" button so that it doesn't slow initial load.

**Comment by peetucket** [comment]
Ben has smoked tested this branch on argo-stage and approved.  My best guess on performance (no hard data though) is that give its just another solr query to run, its probably going to decrease performance in only a marginally linear way - i.e. if 20 facets make the page take 30 seconds to load, then going to 21 facets will make it take an extra second or so.

**Pull request merged**